justfile

```diff
@@ -28,6 +28,9 @@ test *args="//...":
 test-asan *args="//...":
   just test {{args}} --config=asan
 
+test-stream args:
```

Comment:
Can you add an example argument as a comment to here? Should we have `//...` as the default value?

Reply:
I think that would be too slow and too much output - I think it's best when running a single test.

Reply:
Would you mind adding a comment for running an example test?